### ISSUE TITLE
The pages/contribute.md file is not formatted and is not really usable

### ISSUE THREAD

**Issue by bob-fontana** [GSA/ficam-guides#6]
The pages/contribute.md file is not formatted.  It should look like the file in piv-guides.

**Comment by lachellel** [comment]
This is the overarching navigation ...not for specific guides...

I think in relation to: 

#3 - FPKI guides?
#4 and #5 - we need another repo...audiences = Facilities Security Officers, physical access control systems

**Comment by bob-fontana** [comment]
What happened is I opened the contribute.md file to follow the instructions on how to contribute but they were all jumbled up.  I think that the HTML and JavaScript at the top affected the browser's rendering of the instructions on how to contribute.  So I went to piv-guides where CONTRIBUTE.md is nicely formatted.  Because I was in that folder, I ended up mistakenly clicking the Pull Request button there since I was already in there reading the nicely formatted CONTRIBUTE.md.  Fortunately, I realized my mistake and withdrew my PR and re-submitted it under ficam-guides (which still might change).  But, my suggestion is to copy the exact markdown from piv-guides/CONTRIBUTE.md to ficam-guides/pages/contribute.md so that nobody makes the same mistake of looking for a better formatted contribute.md file.
